app/plugin_api.go

```diff
@@ -192,6 +192,16 @@ func (api *PluginAPI) UpdateUserStatus(userId, status string) (*model.Status, *m
 
 	return api.app.GetStatus(userId)
 }
+
+func (api *PluginAPI) GetUsersInChannel(channelId, sortBy string, page, perPage int) ([]*model.User, *model.AppError) {
+	switch sortBy {
+	case "status":
+		return api.app.GetUsersInChannelByStatus(channelId, page*perPage, perPage)
+	default:
```

Comment:
Sorry for the ambiguity in my original comment, but propose adding a `case "default":` here instead, with all unknown `sortBy` parameters (e.g. the actual `default:`) returning an error for now. 0/5 on whether `default` is the right keyword to use or not for this case.

This avoids unexpected surprises when someone calls `GetUsersInChannel(channelId, "userID")` thinking they get that sort behaviour, only for it to change later if we actually add support for same.

Reply:
Actually, looking at the code in the SQLStore, it looks like `GetUsersInChannel` sorts by `username`, so that's probably the best term to use here.

Reply:
When you say "the best term to use here", I'm assuming you mean `case "username"` rather than `case "default"`. Is that correct?

So essentially, the `sortBy` parameter will become required knowledge as an enumerable value and consumers will need to know what values are available.

Reply:
Yes, it definitely makes sense to accept `"username"` explicitly. 

0/5, but we could also accept `""` and handle that the same as `"username"` as a "default" parameter, but then rejecting all other unknown values. But given that the signature requires a `sortBy` parameter, the author has to supply something, and I'm not a huge fan of "mystery" parameters like:
```
p.API.GetUsersInChannelByStatus(channelId, "", 0, 100)
```

This seems better for the plugin author:
```
p.API.GetUsersInChannelByStatus(channelId, "username", 0, 100)
```

We could even throw this as a constant in `model` as a `CHANNEL_SORT_BY_USERNAME`, `CHANNEL_SORT_BY_STATUS` and `CHANNEL_SORT_BY_DEFAULT`.